test_gemini_with_tools.py

```
@@ -0,0 +1,133 @@
+            "temperature": 72,
+            "conditions": "sunny",
+            "forecast": ["sunny", "cloudy", "rainy"]
+        }
+    }
+
+def run_gemini_test():
+    # Try to load from .env file
+    try:
+        load_dotenv()
+    except:
+        pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)